libp2p/Host.cpp

```diff
@@ -280,7 +280,7 @@ void Host::determinePublic(string const& _publicAddress, bool _upnp)
 	{
 		if (!m_peerAddresses.count(reqpublicaddr))
 			m_peerAddresses.insert(reqpublicaddr);
-		m_public = reqpublic;
```

Comment:
This `reqpublic` and `reqpublicaddr` would they not be better as `reqPublic` and `reqPublicAddr` respectively? It was hard for me to read it as is. Thought there was a typo.